### PR TITLE
LPS-148411 [CP 2.0] LES activation - Users message when license not available yet

### PR DESCRIPTION
**Pull request by joaosvassis**
### How it was developed?
 I used graphql to query the user roles in the project to display the messages according to the role, I used useMemo and some for better performance and to facilitate the logic.

### CODE REVIEW

**Review by joanagrego — APPROVED** [review]
Nice job!

**Review by claricebrasil — APPROVED** [review]
Great!

**Review by jairmedeiros — CHANGES_REQUESTED** [review]

File: modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/extra/remote-app/src/routes/customer-portal/layouts/ActivationKeysLayout/Inputs/index.js [hunk 1]
```diff
@@ -132,6 +136,55 @@ const ActivationKeysInputs = ({
 		setLicenseDownloadError(true);
 	};
 
+	const AccountBrief = userAccount.accountBriefs?.find(
```
[the remark this answers is not in the document]
Comment (joaosvassis) [reply modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/extra/remote-app/src/routes/customer-portal/layouts/ActivationKeysLayout/Inputs/index.js]: Ok, thank you!

**Comment by jairmedeiros** [comment]
ci:forward

**Review by igordmf — APPROVED** [review]
Good job!

**Pull request closed without merging**